Create RoleNORManager.md (#2147)
OBOFoundry/OBOFoundry.github.io: docs/roles/RoleNORManager.md

> ## OBO Role: New Ontology Request (NOR) Manager

Reviewer: ```suggestion
---
layout: doc
permalink: /role/nor-manager
title: OBO Role: New Ontology Request (NOR) Manager
---
```

Permalink actually lets you specify what the address is, rather than using the sketchy looking path + name

---
layout: doc

> permalink: /role/nor-manager

Reviewer: ```suggestion
permalink: /roles/nor-manager
```
to go along with https://obofoundry.org/roles/

title: OBO Role: New Ontology Request (NOR) Manager
---

The New Ontology Request (NOR) Manager has these basic duties:

1. Receiving `new ontology` requests on OBO and acknowledging the receipt

> 2. Including the ontology into the OBO New Ontology Request (NOR) Dashboard

Reviewer 2: ```suggestion
2. Including the ontology in the OBO New Ontology Request (NOR) Dashboard
```

> 3. Supporting the New Ontology requestor to passing the OBO Dashboard

Reviewer 2: ```suggestion
3. Supporting the New Ontology requestor to pass the OBO Dashboard
```

4. Assigning a provisional reviewer from the OBO Operations Committee
5. Removing the accepted ontology from the Dashboard upon acceptance

### Receiving `new ontology` requests on OBO and acknowledging the receipt

- The NOR Manager MUST actively listen to issues on the [OBO Issue Tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues).
- When a new ontology request is received, and the wrong issue template was used, the New Ontology Request (NOR) Manager must inform the NOR requestor to use the correct issue template and close the wrongly submitted issue.
- When a valid NOR request is received the NOR Manager acknowledges the receipt with the following message:
    ```markdown
    Dear @GITHUBNAME,
    
    Thank you for your submission. The review will be executed as a two stage process. 
    First, you will have to pass [OBO NOR Dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html). Pass means that no check _apart from `Users` may be red_.
    After you have successfully passed the Dashboard you will be assigned an OBO Operations committee member to review the ontology.
    

>     Usually, the review will result in an opportunity to you to improve the ontology.

Reviewer 2: ```suggestion
    Usually, the review will result in an opportunity for you to improve the ontology.
```

    When the reviewer believes the ontology is ready for presentation to the OBO Operations Committee, they will present
    your ontology during an OBO Operations Call. This gives other members of the
    committee the opportunity to assess your work.
    
    When a decision is reached by the committee you will be informed here on the issue tracker.
    The process can take any number of weeks or months, depending on the case at hand.
    Please inform us about any reasons you might have for increased urgency.
    
    You will be informed once your ontology is loaded in the OBO NOR Dashboard.
    
    Good luck.
    ```

> ### Including the ontology into the OBO New Ontology Request (NOR) Dashboard

Reviewer 2: this header seems misplaced; the section that follows isn't about the dashboard, it's post-dashboard.

Author: I dont know what happened here. Fixed now.

>     After the dashboard is run, the OBO dashboard administrator informs the submitter about the need to fix the issues revealed by the dashboard, noting this is not part of the review itself, just a precursor, and that upon completion, a liaison will be assigned.

Reviewer 2: This suddenly starts using the word "liaison" rather than reviewer.

>     At the next OBO Foundry Operations Committee conference call (hereafter, "Operations call"), a liaison is selected to be responsible for the issue. This liaison becomes familiar with the new ontology and rallies the appropriate people to provide feedback.

Reviewer 2: this seems to be straying from the NOR Manager role and going deep into the ontology review & acceptance process. consider moving the text here and to the end of this document to a more appropriate document.

>     At the next Operations call after that one, the liaison presents the New Ontology Request (NOR) to the OBO Foundry Operations Committee and answers questions. In most cases, the information provided will be sufficient to either grant or refuse the prefix request. In some cases, the committee may choose to postpone its decision to require some clarification and fixes from the submitter. The liaison MUST be present at the Operations call in order for the NOR case to be discussed. If the liaison does not participate for 2 consecutive Operations calls, the chair of the second call emails the liaison to request a statement confirming the ability to continue as liaison. If the liaison does not participate in 3 consecutive Operations calls and did not respond to the email above, a new liaison is assigned during that third call.

Reviewer 2: "grant or refuse the prefix request" - this is the first time prefix is mentioned, and there's no context provided. either rephrase this "the request to be included as an OBO Foundry ontology", or explain prefix requests.

Reviewer 2: (this line is very long; would be easier to comment and make mergeable suggestions if it were shorter)
"In some cases, the committee may choose to postpone its decision to require some clarification and fixes from the submitter." - postpone its decision about acceptance, pending some clarifications or fixes from the submitter.

Reviewer 2: Never got back to final NOR Manager duty, "Removing the accepted ontology from the Dashboard upon acceptance" -- maybe at least state that, even if you don't think any further details are needed?

---
layout: doc
permalink: /roles/nor-manager

> title: OBO Role: New Ontology Request (NOR) Manager

Reviewer: using a colon inside here breaks jekyll

---

The New Ontology Request (NOR) Manager has these basic duties:

1. Receiving `new ontology` requests on OBO and acknowledging the receipt
2. Including the ontology in the OBO New Ontology Request (NOR) Dashboard
3. Supporting the New Ontology requestor to pass the OBO Dashboard
4. Assigning a provisional reviewer from the OBO Operations Committee

> 5. Removing the accepted ontology from the Dashboard upon acceptance

Reviewer 2: ```suggestion
5. Removing the accepted ontology from the NOR Dashboard upon acceptance
```

For many New Ontology requestors it is not always easy to interpret the warnings on the OBO Dashboard. The NOR Manager is responsible for supporting the requestor interpreting the dashboard (but _not_ to fix the problems). There is no need to dig deep into the ontologies to diagnose problems - that is the requestors responsibility - but pointing to documentation like the [OBO Foundry Principles](https://obofoundry.org/principles/fp-000-summary.html) may be necessary.

### Assigning a provisional reviewer from the OBO Operations Committee

If and only if the requested ontology passes the dashboard, i.e. no more "red" exists aside from the `Usage` column, the NOR manager assigns the next available reviewer from the ['OBO Operations Members' sheet in the OBO Duty Rotation Table](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit). The reviewer is assigned by 

1. Adding them to the ['Ontology Review Rotation' sheet in the OBO Duty Rotation Table](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit)
2. Assigning the NOR GitHub issue to the reviewer

> 3. Announcing the assignment to the OBO Operations Technical Working Group Liason to include the the OBO Operations Meeting Report.

Reviewer 2: ```suggestion
3. Announcing the reviewer assignment to the OBO Operations Technical Working Group Liaison to include in their report at the next OBO Operations meeting.
```

> ### Removing the accepted ontology from the Dashboard upon acceptance

Reviewer 2: ```suggestion
### Removing the accepted ontology from the NOR Dashboard upon acceptance
```

> When the ontology is accepted, the OBO NOR review needs to make sure that the accepted ontology is removed from the OBO NOR Dashboard.

Reviewer 2: ```suggestion
When the ontology is accepted, the OBO NOR reviewer needs to make sure that the accepted ontology is removed from the OBO NOR Dashboard.
```
